Resources/public/templates/partials/tree.hbt

```diff
@@ -1,6 +1,6 @@
 <ul class="ez-tree-level">
 {{#each children}}
-    <li class="ez-tree-node{{#if state.leaf}} is-tree-node-leaf{{/if}} {{#if state.open}}is-tree-node-open{{else}}is-tree-node-close{{/if}}" data-node-id="{{id}}">
+    <li class="ez-tree-node{{#if state.leaf}} is-tree-node-leaf{{/if}} {{#if state.open}}is-tree-node-open{{else}}is-tree-node-close{{/if}}" data-node-id="{{id}}" title="{{ data.contentInfo.name }}">
```

Comment:
ping @bdunogier @dpobel 
Like @joaoinacio said this is a case where using text simplify things. What are your opinions on this?